uint64 timestamp # timestamp of when message is sent

float32 x # desired x coordinate 
float32 y # desired y coordinate
float32 z # desired z coordinate
float32 yaw # desired yaw

float32 velocity_x # the desired x velocity 
float32 velocity_y # the desired y velocity 
float32 velocity_z # the desired z velocity 
float32 velocity_yaw # the desired yaw velocity

